internal/provider/data_source_aws_network_peering.go

```diff
@@ -31,6 +32,13 @@ func dataSourceAwsNetworkPeering() *schema.Resource {
 				Required:         true,
 				ValidateDiagFunc: validateSlugID,
 			},
+			// Optional inputs
+			"wait_for_active_state": {
```

Comment:
This is outside vnet peering scope. Can this be in another PR?

Reply:
Good point